AnkiDroid/src/main/java/com/ichi2/anki/Info.java

```diff
@@ -236,6 +247,9 @@ public String copyDebugInfo() {
         return debugInfo;
     }
 
+    private String getWebviewUserAgent() {
+        return new WebView(this).getSettings().getUserAgentString()+ "\n\n";
```

Comment:
```suggestion
        return new WebView(this).getSettings().getUserAgentString() + "\n\n";
```
https://github.com/ankidroid/Anki-Android/wiki/Code-style#white-space-should-be-used-in-the-following-cases